pom.xml

```diff
@@ -72,53 +68,45 @@
   </scm>
 
   <properties>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <jenkins.version>1.554.1</jenkins.version>
```

Comment:
I would not advise even trying to use the new parent with anything older than 1.580.1; to start with, we would not be able to use `jenkins-test-harness` 2.x.

That said, this would be a fine time to increment the dependency IMHO. 1.554.x is now quite old.